src/hotspot/share/gc/g1/heapRegion.cpp

```diff
@@ -714,7 +714,8 @@ void HeapRegion::verify(VerifyOption vo,
     p += obj_size;
   }
 
-  if (!is_empty()) {
+  // only regions in old generation contain valid BOT
```

Comment:
Please start the sentence with upper case and end with full stop.

Reply:
Fixed.